public/docs/ts/latest/guide/server-communication.jade

```diff
@@ -628,6 +629,26 @@ block wikipedia-jsonp+
       You added the `debounceTime`, `distinctUntilChanged`, and `switchMap` operators to the RxJS `Observable` class
       in `rxjs-operators` as [described above](#rxjs).
 
+a#xsrf
+.l-main-section
+:marked
+  ## Guarding against Cross-Site Request Forgery
+
+  In a cross-site request forgery (CSRF or XSRF), an attacker tricks the user into visiting
```

Comment:
Thought about that. We need some context here.  I only repeat the intro and this is material that shouldn't change even if our implementation does. I didn't think making up a new summary description would make a difference.